### ISSUE TITLE
`[ERR_MODULE_NOT_FOUND]: Cannot find module` with exports map

### ISSUE THREAD

**Issue by SettingDust** [node-loaders/loaders#6]
https://github.com/SettingDust/esbuild-esm-loader-exports-map-ts

**Comment by mshima** [comment]
TS only packages is not usual, so it's filtered:
https://github.com/node-loaders/loaders/blob/3375740be1456ec28ff7e3c6acffdbc94dfd062c/workspaces/esbuild/src/esbuild-loader.ts#L30C11-L30C11

**Comment by SettingDust** [comment]
Sorry. I'm not get you. You means the `forwardPackageSpecifiers` filtered ts?
It's usual in monorepo before compile. Such as running tests

**Comment by mshima** [comment]
No, it filters package specifiers. So we don’t look for ts files inside packages by default.
Not sure if we should look ts files at packages by default. Is more likely to be a bug.
An ENV variable is ok?
Do you know if esbuild or other tool accepts it?

**Comment by SettingDust** [comment]
no issue with esbuild build the package.

**Comment by SettingDust** [comment]
> The link doesn't work.

Haven't noticed it's private. I've made it public now XD
